recipes/pytest-split/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pytest-split" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf0163aafe5686e917a7f043567af96c50fabfe1ba0aa3be0f1b65556e09aa3c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6,<4
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.6,<4
+    - pytest >=5.4
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+    - pytest --traceconfig | grep pytest-split-{{ version }}
+
+about:
+  home: https://github.com/henry0312/pytest-pycodestyle
```

Comment:
`home` looks like the incorrect url -- probably a cut and paste mistake from another recipe.

Reply:
Thanks for reviewing it, just fixed it.